dotcom-rendering/src/components/Card/components/LI.tsx

```diff
@@ -113,7 +117,8 @@ export const LI = ({
 						GAP_SIZE,
 						containerPalette,
 					),
-				padSides && sidePaddingStyles(padSidesOnMobile),
+				sidePaddingStyles,
+				padSides && sidePaddingStylesMobile(padSidesOnMobile),
```

Comment:
```suggestion
				padSides && sidePaddingStyles,
				padSidesOnMobile&& sidePaddingStylesMobile,
```

and then neither styles need to be a function!